src/components/wallets/WalletAccordion.tsx

```diff
@@ -0,0 +1,327 @@
+import { useState } from 'react'
+import { Accordion, AccordionSummary, AccordionDetails } from '@mui/material'
+import styled from 'styled-components'
+import { DateTime } from 'luxon'
+
+import { theme } from '~/styles'
+import {
+  Avatar,
+  Button,
+  Icon,
+  Skeleton,
+  Status,
+  StatusEnum,
+  Tooltip,
+  Typography,
+} from '~/components/designSystem'
+import { useInternationalization } from '~/hooks/core/useInternationalization'
+import { Wallet, WalletStatusEnum } from '~/generated/graphql'
+
+interface WalletAccordionProps {
+  wallet: Omit<Wallet, 'updatedAt'>
```

Comment:
I don't get this 🤔 And where is "Wallet" comming from ? Are you using the type generated from the API ?
If the answer is yes, you should use the type generated buy the codegen from the fragment you've declared

Reply:
I can now use the wallet detail Fragment to define the shape here, after the change related to your comment above